extraTs/interfaces.ts

```diff
@@ -0,0 +1,14 @@
+interface Ipermissions {
+  getUsers: Igetusers;
```

Comment:
define the types for the inner object also here itself

Reply:
Done